src/plugins/intel_gpu/src/kernel_selector/core/actual_kernels/non_zero/gather_nonzero_kernel_ref.cpp

```diff
@@ -44,37 +44,15 @@ KernelsData GatherNonzeroKernelRef::GetKernelsData(const Params& params, const o
 
     auto entry_point = GetEntryPoint(kernelName, newParams.layerID, params, options);
     auto cldnn_jit = MakeBaseParamsJitConstants(newParams);
+    cldnn_jit.AddConstant(MakeJitConstant("OV_INPUT_RANK", newParams.ov_input_rank));
+    cldnn_jit.AddConstant(MakeJitConstant("TOTAL_DATA_SIZE", newParams.inputs[0].LogicalSize()));
+    if (newParams.inputs[0].LogicalSize() * newParams.ov_input_rank < (64*1024)/(newParams.outputs[0].ElementSize()))
```

Comment:
please replace magic constant by variable with meaningful name.

Reply:
And define it in kernel via jitter too

Reply:
Fixed it and make it to be read from param.